### PR TITLE
Add CI GitHub Action

### PR DESCRIPTION
**Pull request by GhiocelAndrei**
Solves #1.

### CODE REVIEW

File: IssueTrackerAPI.sln [hunk 1]
```diff
@@ -11,6 +11,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IssueTracker.DataAccess", "
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IssueTracker.Abstractions", "IssueTracker.Abstractions\IssueTracker.Abstractions.csproj", "{E4399919-3974-4BA4-8D2D-C577DACE38B2}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".github", ".github", "{7389B285-07B3-4B4C-91D3-12B70A46FBC3}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "workflows", "workflows", "{BACBC9E6-604F-44D8-A1CB-138EE2BA1B0B}"
+	ProjectSection(SolutionItems) = preProject
+		testing.yml = testing.yml
+	EndProjectSection
+EndProject
```
Comment (cristi-paraschiv) [inline IssueTrackerAPI.sln:20]: why are these changes needed?

Comment (GhiocelAndrei) [reply IssueTrackerAPI.sln]: I removed them in the latest commit

**Review by cristi-paraschiv — APPROVED** [review]
